File: src/app/core/types/video.ts
export class Video {
  id: number;
  title: string;
  image: string;
  imageCrop: string;
  imageExtension: string;
  imageUrl: string;
  imageCropUrl: string;
  url: string|any;

  constructor(id: number, title: string, image: string, imageCrop: string, imageExtension: string,
    imageUrl: string, imageCropUrl: string, url: string) {
    this.id = id;
    this.title = title;
    this.image = image;
    this.imageCrop = imageCrop;
    this.imageExtension = imageExtension;
    this.imageUrl = imageUrl;
    this.imageCropUrl = imageCropUrl;
    this.url = url;
  }
}

File: backend/Marbomil.Infrastructure/Entities/Database/Video.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Marbomil.Infrastructure.Entities.Database
{
    public class Video : IDbEntity
    {
        public int Id { get; set; }

        [MaxLength(100)]
        public string Title { get; set; }

        [MaxLength(1000)]
        public string ImageUrl { get; set; }

        [MaxLength(1000)]
        public string ImageCropUrl { get; set; }

        [MaxLength(1000)]
        public string Url { get; set; }
    }
}
